sirepo/job.py

```diff
@@ -51,6 +51,9 @@
 #: where user lib file directories are linked for static download (job_supervisor)
 LIB_FILE_ROOT = None
 
+# POSIT: These are the same queues as in schema-common.common.enum.SbatchQueue
+SBATCH_QUEUES = ('debug', 'premium', 'realtime', 'regular')
```

Comment:
I think we should call this NERSC, not SBATCH.  Probably should be a frozenset.

Reply:
https://github.com/radiasoft/sirepo/pull/2511/commits/779ddedc7bc17f3c385612be4fee7c6b6a503204